airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -3,6 +3,13 @@
         "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
         "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
 <module name="Checker">
+    <!-- TODO: Remove this supression once RegexpMultiline has been replaced with an AST aware check -->
+    <!-- This suprpresses Checkstyle between onccurences of """", i.e. text blocks -->
+    <module name="SuppressWithPlainTextCommentFilter">
+        <property name="offCommentFormat" value='=\s+"""'/>
+        <property name="onCommentFormat" value='^\s+.*""";'/>
```

Comment:
Take into consideration the following use case:
- the text block is a parameter among other parameters to a method call (can be followed by `,` or `)`

Reply:
Hmm, that makes it slightly challenging. Since we need some way to differentiate opening and closing text block markers. One way to do this is to enforce all text-block content appears on a newline by changing start marker to `\s+"""$`.

Changing closing marker to `^\s+.*""".+` would mean it can still match opening marker if we don't enforce that text blocks start on a new line.

Any suggestions?
In meantime I'm looking at replacing RegexpMultiline.

Reply:
I guess your solution is based on the proposal

https://github.com/checkstyle/checkstyle/issues/11653#issuecomment-1180373074

What if we use something like:

```
<property name="onCommentFormat" value='^\s+.*"""(;|,|\))'/>
```

?

Reply:
That covers most of the cases we know of. Thanks. Hopefully this is a short term solution so we don't need to make it pretty.

Reply:
There are a few minor variations when dealing with `)`, it can be followed optionally by other `)` and eventually by `,` or `;`. Good thing is that AFAIK they are all on the same line.

```
<property name="onCommentFormat" value='^\s+.*"""(,|;|(\)+(,|;)))'/>
```

NOTE that I tested this so far only on regex101.com and not with the changes on your branch.

Reply:
> * the text block is a parameter among other parameters to a method call (can be followed by `,` or `)`

i think this is important.
It's confusing to allow 

```
String x = """
   )
 """;
```

while not allow
```
assertQuery(
   """
    )
    """);
```

the text blocks are awesome for SQL queries and SQL queries are awesome to inline in tests, so this would be primary use-case